frontend/tests/unit/components/attendance-list-test.js

```
@@ -86,24 +86,18 @@ test('it lists todays attendances', function(assert) {
 })
 
 test('it marks activities to review with a warning class', function(assert) {
-  assert.expect(2)
+  assert.expect(1)
```

You can use skip() instead of test() to skip failing tests